apps/deploy-web/tests/pages/DeployBasePage.tsx

```diff
@@ -30,7 +31,33 @@ export class DeployBasePage {
     };
   }
 
-  async submit() {
+  async createDeployment() {
     await this.page.getByTestId("create-deployment-btn").click();
+    await this.page.getByTestId("deposit-modal-continue-button").click();
+  }
+
+  async createLease() {
+    await this.page.getByTestId("create-lease-filter-audited").click();
+    await this.page.getByTestId("bid-list-row-radio-0").click();
+    await this.page.getByTestId("create-lease-button").click();
+  }
+
+  async validateLease() {
+    await this.page.waitForURL(/http:\/\/localhost:3000\/deployments\/\d+/);
```

Comment:
should port be an env var?

Reply:
actually yea the url should be an env var if we run the tests on staging or other live env.